src/rqpoly.rs

```diff
@@ -252,8 +266,23 @@ where
         c.inverse_transform();
         c
     }
+
+    fn coeffwise_multiply(&self, other: &Self) -> Self {
+        let context = self.context.as_ref().unwrap();
+        let mut c = self.clone();
```

Comment:
maybe you want a `zip_eq` instead of `zip` here?

Reply:
Good idea! It seems doing so would need `itertools` as a dependency tho. Since we wish to minimize the num of dependencies, maybe we leave it as-is for now?